src/sas/qtgui/Calculators/DataOperationUtilityPanel.py

```diff
@@ -141,15 +145,31 @@ def onCompute(self):
 
         self.output = output
 
+        self.updatePlot(self.graphOutput, self.layoutOutput, self.output, color='#000000',
+                        operation_data=True, data_op=[self.data1, self.data2], color_op=["#44AA99", "#CC6677"],
```

Comment:
we already define colours for BG_RED and BG_WHITE as module variables.
Please consider doing the same for the numerical RGB values - those don't "feel" nice.

Reply:
The plotting colors have been set as module variables.